README.md

```diff
@@ -0,0 +1,342 @@
+# TencentCloud MySQL Module for Terraform
+
+## terraform-tencentcloud-mysql
+
+A terraform module used to create TencentCloud MySQL instance, readonly instances and database privileges.
+
+The following resources are included.
+
+* [MySQL](https://www.terraform.io/docs/providers/tencentcloud/r/mysql_instance.html)
+* [MySQL Account](https://www.terraform.io/docs/providers/tencentcloud/r/mysql_account.html)
+* [MySQL Account Privilege](https://www.terraform.io/docs/providers/tencentcloud/r/mysql_account_privilege.html)
+* [MySQL Backup Policy](https://www.terraform.io/docs/providers/tencentcloud/r/mysql_backup_policy.html)
+* [MySQL Privilege](https://www.terraform.io/docs/providers/tencentcloud/r/mysql_privilege.html)
+* [MySQL Readonly Instance](https://www.terraform.io/docs/providers/tencentcloud/r/mysql_readonly_instance.html)
+
+## Usage
+
+For new instance
+
+```hcl
+module "mysql" {
+    source = "terraform-tencentcloud-modules/mysql/tencentcloud"
+
+    ###############
+    # MySQL Instance
+    ###############
+
+    instance_name = "5.7"
+    mem_size = 1000
+    volume_size = 200
+    root_password = "root_passwd1"
+    availability_zone = "ap-guangzhou-1"
+    security_groups = ["sg-nltpbqg1"]
+    vpc_id = "vpc-h70b6b49"
+    subnet_id = "subnet-pyio7yog"
+    first_slave_zone = "ap-guangzhou-1"
+
+    tags = {
+        Env = "dev"
+    }
+
+    ###############
+    # MySQL Account
+    ###############
+
+    accunt = [
```

Comment:
Fix the typos and this request can be approved. (There are 4 `accunt` typo in this README)

Reply:
fixed, thanks